assembly/collections/persistentMap.ts

```diff
@@ -43,16 +43,16 @@ export class PersistentMap<K, V> {
    * ```ts
    * let map = new PersistentMap<string, string>("m") // note the prefix must be unique (per NEAR account)
    * ```
-   * @param {string} prefix A prefix to use for every key of this map.
+   * @param prefix - A prefix to use for every key of this map.
    */
   constructor(prefix: string) {
     this._elementPrefix = prefix + _KEY_ELEMENT_SUFFIX;
     this._size = 0;
   }
 
   /**
-   * @param {string} key - Search key.
-   * @return {string} An internal string key for a given key of type K.
+   * @param key - Search key.
```

Comment:
why mixing @param and @typeParam ?

Reply:
typeParam is for generic type https://typedoc.org/tags/typeParam/

Reply:
The @typeParam tag is used to document a type parameter of a function, method, class, interface or type alias.